### PR TITLE
Instance change username and ownership

### PR DESCRIPTION
**Pull request by Mariana-na**

### CODE REVIEW

File: docs/manage-cloud/cloud-data-management.md
Comment (ayatnkw) [inline docs/manage-cloud/cloud-data-management.md:1]: @Mariana-na did you want to include this as part of this PR?

**Review by ayatnkw — COMMENTED** [review]
Could you add this new page to the mkdocs.yml page so that it shows in the sidebar?

File: docs/manage-cloud/cloud-data-management.md
Comment (Mariana-na) [reply docs/manage-cloud/cloud-data-management.md:1]: Oh no, I had some issues getting my local repo up to date and must've made a mess, I'm sorry!
How do you want me to proceed?

Comment (ayatnkw) [reply docs/manage-cloud/cloud-data-management.md:1]: Could you remove this change from the PR? That would be great :)

File: docs/changing-ownership/change-ownership-or-username.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Change Instance Ownership or Username
+Description: Change instance ownership or username.
+contentType: howto
+---
+
+# Change instance ownership
+
+You can change the ownership of an instance by navigating into the **Settings** page in the owner's account and editing the **Email** field. After making the changes, scroll down and press **Save**.
+Note that for the change to be effective, the new email address cannot be linked to any other n8n account.
```
Comment (ayatnkw) [inline docs/changing-ownership/change-ownership-or-username.md:11]:
```suggestion
For the change to be effective, you can't link the new email address to any other n8n account.
```

File: docs/changing-ownership/change-ownership-or-username.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Changing emails will change the owner of the instance, the email you log in with, and the email your invoices and general communication gets sent to.
+
+# Change instance username
+
+It is not currently possible to change usernames.
```
Comment (ayatnkw) [inline docs/changing-ownership/change-ownership-or-username.md:17]:
```suggestion
It's not currently possible to change usernames.
```

File: docs/changing-ownership/change-ownership-or-username.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Change Instance Ownership or Username
+Description: Change instance ownership or username.
+contentType: howto
+---
+
+# Change instance ownership
+
+You can change the ownership of an instance by navigating into the **Settings** page in the owner's account and editing the **Email** field. After making the changes, scroll down and press **Save**.
```
Comment (ayatnkw) [inline docs/changing-ownership/change-ownership-or-username.md:10]:
```suggestion
You can change the ownership of an instance by navigating to the **Settings** page in the owner's account and editing the **Email** field. After making the changes, scroll down and press **Save**.
```

File: docs/changing-ownership/change-ownership-or-username.md (hunk 1 above)
Comment (ayatnkw) [inline docs/changing-ownership/change-ownership-or-username.md:11]:
```suggestion
Note that for the change to be effective, the new email address can't be linked to any other n8n account.
```

File: docs/changing-ownership/change-ownership-or-username.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+Note that for the change to be effective, the new email address cannot be linked to any other n8n account.
+
+Changing emails will change the owner of the instance, the email you log in with, and the email your invoices and general communication gets sent to.
+
+# Change instance username
+
+It is not currently possible to change usernames.
+If you want your instance to have a different name you need to create a new account and transfer your work into it.
```
Comment (ayatnkw) [inline docs/changing-ownership/change-ownership-or-username.md:18]: Maybe explain how they can transfer work into the new account by linking this page: https://docs.n8n.io/workflows/export-import/ ?

Comment (Mariana-na) [reply docs/changing-ownership/change-ownership-or-username.md:18]: great point!

**Review by ayatnkw — APPROVED** [review]

**Pull request merged**